packages/animation/install

```diff
@@ -4,3 +4,4 @@ set -e
 
 apt-get update -qq
 apt-get install -y graphicsmagick libav-tools
+apt-get install -y pdftk
```

Comment:
Please just add to above line

Reply:
so I shall remove the tests?

Reply:
Nope, test is great.
Just do the `apt-get install` as one call

Reply:
aah ok